src/main/java/seedu/address/model/entity/body/Relationship.java

```diff
@@ -0,0 +1,12 @@
+package seedu.address.model.entity.body;
+
+//@@author ambervoong
+/**
+ * Represents all the common familial relationships.
+ */
+public enum Relationship {
+    HUSBAND, WIFE, FATHER, MOTHER, STEPFATHER, STEPMOTHER, GUARDIAN, SON, DAUGHTER, STEPSON,
```

Comment:
Should we add in "Friend"

Reply:
Maybe we can consider not making it an enum since there are so many choices. Up to you!

Reply:
Oh yikes, yes definitely we should add "friend". It's true about the enum, most of our fields are really broad. I will change it then.

Reply:
Done.